pkg/collector/corechecks/snmp/checkconfig/config_metric.go

```diff
@@ -178,7 +178,7 @@ func (mtc *MetricTagConfig) GetTags(value string) []string {
 				normalizedTemplate := normalizeRegexReplaceValue(val)
 				replacedVal := RegexReplaceValue(value, mtc.pattern, normalizedTemplate)
 				if replacedVal == "" {
-					log.Debugf("pattern `%v` failed to match `%v` with template `%v`", value, normalizedTemplate)
+					log.Debugf("pattern `%v` failed to match `%v` with template `%v`", value, mtc.pattern, normalizedTemplate)
```

Comment:
```suggestion
					log.Debugf("pattern `%v` failed to match `%v` with template `%v`", mtc.pattern, value, normalizedTemplate)
```

Reply:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Should this be 
```suggestion
					log.Debugf("value `%v` failed to match pattern `%v` with template `%v`", value, mtc.pattern, normalizedTemplate)
```
?